src/EncompassRest/Contacts/BusinessContacts.cs

```diff
@@ -0,0 +1,161 @@
+﻿using EncompassRest.Utilities;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace EncompassRest.Contacts
+{
+    public sealed class BusinessContacts
+    {
+        private const string s_apiPath = "encompass/v1/BusinessContacts";
+
+        #region Public Properties
+        public EncompassRestClient Client { get; }
+        #endregion
+
+        internal BusinessContacts(EncompassRestClient client)
+        {
+            Client = client;
+        }
+
+        public Task<BusinessContact> GetContactAsync(string contactId) => GetContactAsync(contactId, CancellationToken.None);
+
+        public Task<BusinessContact> GetContactAsync(string contactId, CancellationToken cancellationToken) =>
+            GetContactInternalAsync(contactId, cancellationToken, response => response.Content.ReadAsAsync<BusinessContact>());
```

Comment:
Need to initialize with `Client` and set `BusinessContact.Dirty` to `false`.

Reply:
like this : 

public Task<BusinessContact> GetContactAsync(string contactId, CancellationToken cancellationToken) =>
            GetContactInternalAsync(contactId, cancellationToken, async response =>
            {
                var contact = await response.Content.ReadAsAsync<BusinessContact>().ConfigureAwait(false);
                contact.Initialize(Client);
                return contact;
            });